data/yaml/programs/graduate.yml

```diff
@@ -476,6 +897,18 @@
   tags:
     - architecture
     - design
+    - community*
+    - creative expression
+    - innovation
+    - neighbourhood
+    - physical design
+    - recreation
+    - resillien*
```

Comment:
If this is referring to resilience, then it's misspelled (should be one L instead of two)